test/cppsuite/test_harness/workload/database_model.h

```diff
@@ -56,6 +56,42 @@ struct collection_t {
 /* Representation of the collections in memory. */
 class database {
     public:
+    /*
+     * Add a new collection following the standard naming pattern. Currently this is the only way to
+     * add collections which is supported by all components, go outside of this model at your own
+     * peril.
+     */
+    std::string
+    add_collection()
+    {
+        std::lock_guard<std::mutex> lg(_mtx);
+        std::string collection_name = build_collection_name(_collection_count);
+        testutil_assert(_collections.find(collection_name) == _collections.end());
+        _collections[collection_name] = {};
+        ++_collection_count;
+        return (collection_name);
+    }
+
+    /*
+     * A not locked way to retrieve the current collection count, may return stale data.
+     */
+    uint64_t
+    get_collection_count() const
+    {
+        return (_collection_count);
+    }
+
+    /*
+     * Get a single collection name by id.
+     */
+    std::string
+    get_collection_name(uint64_t id)
+    {
+        if (_collection_count <= id)
+            testutil_die(-1, "requested the name of a collection that doesn't exist");
+        return build_collection_name(id);
```

Comment:
Add brackets around the return statement

Reply:
f6b340f